frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -232,6 +233,7 @@ export class PieceEditor implements OnInit {
     this.prevWorkingState = undefined;
     const newPiece: WorkingPiece = {
       uuid: 'segment:' + Date.now(),
+      isOpen: type === 'REVELATION' ? true : false,
```

Comment:
Please add a comment for why this should be true when type is REVELATION

Reply:
Done.